src/appReview/saga.ts

```diff
@@ -0,0 +1,47 @@
+import InAppReview from 'react-native-in-app-review'
+import { call, put, select, spawn, takeLatest, takeLeading } from 'redux-saga/effects'
+import { appReviewSelector } from 'src/appReview/selectors'
+import { setInitialized, setLastInteractionTimestamp } from 'src/appReview/slice'
+import { Actions as SendActions } from 'src/send/actions'
+import { safely } from 'src/utils/safely'
+import { ONE_DAY_IN_MILLIS } from 'src/utils/time'
+import { Actions as Web3Actions } from 'src/web3/actions'
+
+export function* initAppReview() {
```

Comment:
i elaborated on this in my other comment but i still don't understand the value of this saga, can you explain what fails if we delete this?